src/Components/ErrorPage/ErrorPage.jsx

```diff
@@ -0,0 +1,20 @@
+import "./styles.css";
+
+export const ErrorPage = () => {
+	return (
+		<div className="error-page-container">
+			<div className="error-page-main">
+				<img
+					src="https://i.postimg.cc/K8mHKTz6/dark-Error-Page.png"
+					alt="404-img"
+					className="error-hero"
+				/>
+				<h1 className="error-page-head">Page Not Found</h1>
+				<h2 className="error-page-text">
```

Comment:
As this is a not a heading, or a subheading, you can use a <p> tag or some other appropriate semantic html tag for it. :D